ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.h

```diff
@@ -88,7 +92,7 @@ NS_ASSUME_NONNULL_BEGIN
  multiple plugins.
  */
 
-+ (void)startWithConfigurations:(id<ElectrodePluginConfig>)reactContainerConfig
++ (void)startWithConfigurations:(id<ElectrodePluginConfig>)reactContainerConfig ernDelegate: (id<ERNDelegate>) ernDelegate
```

Comment:
This would result in a breaking change. Can we deprecate the old method and add a new one instead ?

Reply:
Yes, thank you.